patches/tModLoader/Terraria/Terraria.csproj.patch

```diff
@@ -4,18 +4,20 @@
      <Company>Re-Logic</Company>
      <Copyright>Copyright © 2020 Re-Logic</Copyright>
      <RootNamespace>Terraria</RootNamespace>
--	<OutputName>Terraria</OutputName>
-+	<OutputName>tModLoader</OutputName>
-+	<LangVersion>8.0</LangVersion>
+-    <OutputName>Terraria</OutputName>
++    <OutputName>tModLoader</OutputName>
++    <LangVersion>latest</LangVersion>
    </PropertyGroup>
    <PropertyGroup Condition="$(Configuration.Contains('Server'))">
      <OutputType>Exe</OutputType>
-@@ -18,15 +_,23 @@
+@@ -18,15 +_,25 @@
    </PropertyGroup>
    <PropertyGroup>
      <PdbFile>bin\$(OutputName)</PdbFile>
 -    <ApplicationIcon>Icon.ico</ApplicationIcon>
 +    <ApplicationIcon>tModLoader.ico</ApplicationIcon>
++    <AssemblyName>$(OutputName)</AssemblyName>
++    <TargetName>$(OutputName)</TargetName>
```

Comment:
do we need changes to this project?

Reply:
If I remember correctly, (at least some of) these changes are required for .NET Core to resolve the assembly from running tModLoader.exe.